reproman/interface/jobs.py

```diff
@@ -97,11 +98,16 @@ def show_oneline(job, status=False):
     """
     fmt = "{status}{j[_jobid]} on {j[resource_name]} via {j[submitter]}$ {cmd}"
     if status:
-        orc = _resurrect_orc(job)
-        orc_status = orc.status
-        _, queried_status = orc.submitter.status
-        if orc_status == queried_status:
-            # Drop repeated status (e.g., our and condor's "running").
+        try:
+            orc = _resurrect_orc(job)
+            orc_status = orc.status
+            _, queried_status = orc.submitter.status
+            if orc_status == queried_status:
+                # Drop repeated status (e.g., our and condor's "running").
+                queried_status = None
+        except Exception as exc:
```

Comment:
As the failing tests indicate, this interferes with the expected error handling in the outer level.  So I think the options are (1) broaden the existing handling to catch either the `MissingExternalDependency` you were originally interested in (my preference) or a blanket exception or (2) more comprehensively rework the handling around orc resurrection/status query, adjust the tests, maybe rethink the outer handling.  I don't see a big advantage of the latter, so I'd lean to the narrow variant of 1.

If 2 is the choice, `show` should also be considered.